skimage/restoration/_denoise.py

```diff
@@ -110,10 +110,13 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
     sigma_color : float
         Standard deviation for grayvalue/color distance (radiometric
         similarity). A larger value results in averaging of pixels with larger
-        radiometric differences. Note, that the image will be converted using
-        the `img_as_float` function and thus the standard deviation is in
-        respect to the range ``[0, 1]``. If the value is ``None`` the standard
-        deviation of the ``image`` will be used.
+        radiometric differences. If ``None``, the standard deviation of
+        ``image`` will be used. Note that, if the image is
+        any `int` dtype, ``image`` will be converted using the `img_as_float`
+        function and thus the standard deviation is in respect to the
```

Comment:
```suggestion
        function and thus the standard deviation will be in
```
(future tense to be consistent with the rest of the sentence)